pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -665,6 +668,9 @@ class Build {
 
         context.node('built-in || master') {
             context.stage("sign installer") {
+                // Ensure master context workspace is clean of any previous archives
+                context.sh "rm -f workspace/target/* || true"
```

Comment:
`|| true` is pointless when running with `-f` so I would suggest removing it
```suggestion
                context.sh "rm -f workspace/target/*"
```

Reply:
@sxa i'm not sure it is, if there are no files then the exit code will be set to "1" on the rm -f, causing stage failure ?